Updates to readme, pip info

swernli opened the pull request.

billti commented on README.md:
```diff
@@ -1,6 +1,6 @@
-# Q# language and tooling
+# Modern QDK
```
I think the term 'Modern' will be short lived, and fade into obscurity once the 'classic' QDK is forgotten about. I'd go with a heading of just 'Azure Quantum Development Kit'.

billti commented on vscode/README.md:
```diff
@@ -1,12 +1,10 @@
 
-_Note: To install pre-release versions of this extension, you must click the drop-down
-next to the 'Install' button and select 'Install Pre-Release Version'_
+You can also try out this extension in VS Code for Web at [vscode.dev/quantum](http://vscode.dev/quantum).
```
Should probably be an https link, not http :-)

billti commented on vscode/README.md:
```diff
@@ -1,12 +1,10 @@
+You can also try out this extension in VS Code for Web at [vscode.dev/quantum](http://vscode.dev/quantum).
 
-This extension brings rich Q# language support to VS Code. Develop, build, and run your Q# code from VS Code either locally on simulators, or by submitting a job to Azure Quantum.
+For more information about the Modern QDK and Azure Quantum, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```
Does this duplicate/conflict with the info on line 24 below

swernli replied:
Good catch!

billti commented on vscode/README-DEV.md:
```diff
@@ -5,3 +5,5 @@ of upcoming features.
 To install the tested version of this extension, visit <https://marketplace.visualstudio.com/items?itemName=quantum.qsharp-lang-vscode>.
 The README for the stable build contains details on features and usage.
+
+For more information about the Modern QDK and Azure Quantum, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```
I'd leave this readme minimal and referring to the stable extension for more info (as it does already).

billti reviewed (changes requested):
Update line 3 of vscode/package.json to remove the word 'Preview' from the displayName also.

I'd limit use of the term 'modern' to where we're trying to disambiguate versions. I'm hoping this term is temporary until we the 'classic' QDK is a distant memory.

swernli commented:
> Update line 3 of vscode/package.json to remove the word 'Preview' from the displayName also.
> 
> 
> 
> I'd limit use of the term 'modern' to where we're trying to disambiguate versions. I'm hoping this term is temporary until we the 'classic' QDK is a distant memory.

Fixed the missing name change, and updated a url in that same file. 

I took out a few of the "Modern" usages, but we'll have to see what others think, since it seems there's some effort being put into making that moniker stick.

minestarks commented on pip/README.md:
```diff
@@ -41,7 +41,7 @@ BellState()
 ## Support
 
-For more documentation and to browse issues, please visit the Q# project wiki at [https://github.com/microsoft/qsharp/wiki].
+For more information about the Modern QDK and Azure Quantum, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```
Do we want the word "modern" here?

```suggestion
For more information about the Azure Quantum Development Kit, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```

swernli replied:
a PM might say yes... but I don't feel too strongly. Saves us later updates if/when we want to reduce usage of "Modern"
